fastlane/metadata/android/es-ES/title.txt

```diff
@@ -1 +1 @@
-Administrador de Volumen de Bluetooth
\ No newline at end of file
+Administrador del volumen Bluetooth
```

Comment:
Title has a limit of 30 characters. Do you have a good idea for a shorter name?

Reply:
Good afternoon:

Now that you mention it..........

Well, yes ;)

Gestión del volumen Bluetooth
Gestiona el volumen Bluetooth
Maneja el volumen Bluetooth
Administra el volumen Bluetooth
Administrador de volumen Bluetooth

A hug

Reply:
What is your favorite?

Reply:
Good afternoon @d4rken

Sorry for not telling you which one I would put.

I would put:

Gestión del volumen Bluetooth

Sorry for the mistake of not telling you which one fits best. I don't know where I left my head...

A hug